apps/test/netsim/NetSimTable.js

```diff
@@ -186,7 +188,7 @@ describe("NetSimTable", function () {
     assertEqual(apiTable.log(), '');
 
     // Until poll interval has passed.
-    netsimTable.lastFullUpdateTime_ = Date.now() - (netsimTable.pollingInterval_ + 1);
+    netsimTable.lastRefreshTime_ = Date.now() - (netsimTable.pollingInterval_ + 1);
```

Comment:
The check in `NetSimTable.tick` is
```javascript
if (now - this.lastRefreshTime_ > this.pollingInterval_) {
```
In you example, that evaluates as `10 - 9 > 1 // false` which doesn't trigger a refresh.
Maybe I should modify the test to call out that we don't poll until we're past the interval.